deploy/charts/rook-ceph-cluster/values.yaml

```
@@ -24,6 +24,13 @@ toolbox:
   image: rook/ceph:VERSION
   tolerations: []
   affinity: {}
+  resources: {}
+  # limits:
+  #   cpu: "500m"
+  #   memory: "1024Mi"
+  # requests:
+  #   cpu: "500m"
```

Since the toolbox is usually such low impact or used infrequently, even lower requests might make sense.
```suggestion
  #   cpu: "100m"
```

---

sure, I've updated